### ISSUE TITLE
feat req: Optional desktop buffer / recordings saved to "Desktop" folder

### ISSUE THREAD

**Issue by HeartbeatingForCenturies** [EZ64cool/obs-hadowplay#39]
I often use the replay buffer to capture desktop when I'm on discord w friends / using non fullscreen applications but the plugin auto disables the buffer once the Display Capture is in use which I find kind of frustrating is there any way we can get this added as an optional feature

**Comment by EZ64cool** [comment]
Sounds possible but there might be some nicer ways of doing this.

What are the goals here?

- Do you want the replay buffer to be reset once a game-capture is no longer found?
  * Saved footage will start at the earliest when you closed a fullscreen app
  * Opening a fullscreen app will also then reset the buffer
- Do you want the plugin to organise unrecognised recordings into a folder of your choosing?
  * This would mean you can just disable the Automatic replay buffer and just let the plugin organise your persistent replay buffer

**Comment by FunnyDarkly** [comment]
I guess HeartbeatingForCenturies assumed you second implementation which is "Do you want the plugin to organise unrecognised recordings into a folder of your choosing?".

Shadowplay sorts recording by games name except if there's no active game window. If so Shadowplay puts recording into "Desktop" folder no matter was it Discord, browser or some other software. And also in Shadowplay user should allow to record desktop via checkbox.

So, to sum up, idea here to have separate folder for unsorted recordings. For example, I have such structure within my folder:
-Movies
--Apex Legends
--For Honor
--Elden Ring
--Desktop

And I wanted to save my desktop, discord and browser not to "Movies" folder directly, but to "Movies/Desktop".
Although, this function doesn't as vital for me and I'm just clarifying HeartbeatingForCenturies suggestion.
Have a good day!
